### DISCUSSION TITLE
[Feature] Toggle DAVx5 collections

### DISCUSSION THREAD

**Discussion by Dagda47** [TechbeeAT/jtxBoard#694] [Ideas]
**Is your feature request related to a problem? Please describe.**
I have multiple caldav servers synchronised vía DAVx5 on my smartphone. But i only want specific collections and not all of them show up in jtxBoard.

**Describe the solution you'd like**
Under collections i want the option to toggle specific collections on and off for usage in jtxboard.

**Describe alternatives you've considered**
Removing caldav servers from DAVx5 is not an option because i need all the data for my calendar.

**Comment by patrickunterwegs** [comment]
Hey@Dagda47 ,
Thanks, I'll have a look at this soon!

**Comment by patrickunterwegs** [comment]
Hello @Dagda47 , 
I thought about this again, the consequence of such a flag would be that the entries don't show in the list. 
But you could as well just filter the relevant collections in the list and keep the filter. Is there a reason why this is not an option?

**Comment by Dagda47** [comment]
> Hello @Dagda47 , I thought about this again, the consequence of such a flag would be that the entries don't show in the list. But you could as well just filter the relevant collections in the list and keep the filter. Is there a reason why this is not an option?

That is actually what i tried. Sorry, that i didn't mention it before. Unfortunatly the filter doesn't effect notifications so i get them unfiltered and many unwanted ones.
In fact i want that unflagged collections wouldn't show up in my lists without any filters. So this consequence would be fine for my use case.

**Comment by patrickunterwegs** [comment]
I thought about this again and in fact it doesn't make a lot of sense to implement this in jtx Board. If the goal is to not show the entries and to not apply any logic, there is no point in having the entries at all. So this is something that should be solved in DAVx5. 
Maybe it would be interesting to replace the global selector for the tasks app by a per-collection-preference - ie an option per collection to choose to which app a collection should be synchronized? What do you think @rfc2822 @devvv4ever

**Comment by patrickunterwegs** [comment]
As I don't see this as an implementation in jtx Board, I'll move it to discussions, maybe @rfc2822  and @devvv4ever  still would like to give their opinion on this.

**Comment by rfc2822** [comment]
@Dagda47 You can choose which collections to synchronize in DAVx5. Where's the problem?

If you want to separate tasks and calendars, you can create separate calendar and task collections and then only synchronize what you need.

**Reply by Dagda47** [reply]
Thanks for your reply.

I'm using [posteo](https://posteo.de/en) to host my calendar, tasks and addressbook. Additional collections would cost an additional monthly fee and bloat calendar lists for example in thunderbird.
Therefor it is not really an option for me to duplicate all my family collections.

In the older [tasks app](https://f-droid.org/en/packages/org.dmfs.tasks/), which i used before, i could just tick wanted collections to synchronize in the task app itself. That functionality seems to be lost.

**Reply by rfc2822** [reply]
> In the older [tasks app](https://f-droid.org/en/packages/org.dmfs.tasks/), which i used before, i could just tick wanted collections to synchronize in the task app itself. That functionality seems to be lost.

The collections were always synchronized but maybe not shown in the Tasks app.

At the moment we (DAVx5) don't plan to differentiate between "sync events" and "sync tasks" for collections.

**Reply by Dagda47** [reply]
> > In the older [tasks app](https://f-droid.org/en/packages/org.dmfs.tasks/), which i used before, i could just tick wanted collections to synchronize in the task app itself. That functionality seems to be lost.
> 
> The collections were always synchronized but maybe not shown in the Tasks app.

From the users side of view there were two different options to choose collections to show up in the Tasks app. Visible collections (left picture: "Sichtbare Listen") and synchronized collections (right picture: "Synchonisierte Listen"). Only synchronized collections would actually trigger notifications. I can't really comment what is happening in the background though.

I can understand why you wouldn't undergo such massive changes. Thank you nevertheless for maintaining these wonderful apps!

![2023-07-22_12-34](https://github.com/TechbeeAT/jtxBoard/assets/24296416/341f7554-4fd4-4ba3-a230-96f5940275f0) ![2023-07-22_12-35](https://github.com/TechbeeAT/jtxBoard/assets/24296416/8d299fe3-4118-4c41-8cff-ab33daef3a46)
